sec1_intro_and_setup/test_app/config/environments/development.rb

```diff
@@ -0,0 +1,76 @@
+require "active_support/core_ext/integer/time"
```

Comment:
Whats this file for?

Reply:
The `development` environment is used on your local computer as you interact manually with the application.

Reply:
Your answer doesn't explain the usage of this file.

Reply:
This file is used to manage your development/local computer settings in an easy and usable manner.